code/obj/item/clothing/hats.dm

```diff
@@ -1212,3 +1212,13 @@
 	desc = "The Chief of Cleaning, the Superintendent of Scrubbing, whatever you call yourself, you know how to make those tiles shine. Good job."
 	icon_state = "janitorberet"
 	item_state = "janitorberet"
+
+/obj/item/clothing/head/antlers
+	name = "antlers"
+	desc = "Be a deer and wear these, won't you?"
+	icon = 'icons/obj/clothing/item_ears.dmi'
```

Comment:
~Should probably put it in `item_hats.dmi` (which would mean you don't need this line as that's the default for `/obj/item/clothing/head`?~

EDIT: Oh, I see that you were probably copying cat ears? Okay, that's fine.